### DISCUSSION TITLE
Welcome to siteproxy Discussions!

### DISCUSSION THREAD

**Discussion by netptop** [netptop/siteproxy#70] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rifting** [comment]
why is the site obfuscated?

**Reply by netptop** [reply]
so that we can mitigate middle man attacks.

**Reply by netptop** [reply]
since we now support github and telegram login, and will add more login supports in future .
